version-catalog/src/main/kotlin/io/spine/internal/catalog/entries/ApacheHttp.kt

```diff
@@ -24,11 +24,15 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.spine.internal.dependency
+package io.spine.internal.catalog.entries
```

Comment:
Can we please avoid having plural nouns as the package names? Let's rename it to just `... .entry`.